platform/macos/src/MGLMapViewDelegate.h

```diff
@@ -333,6 +333,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable NSViewController *)mapView:(MGLMapView *)mapView calloutViewControllerForAnnotation:(id <MGLAnnotation>)annotation;
 
+/**
```

Comment:
This method appears under the “Managing Callout Popovers” heading in the header and in documentation:

https://github.com/mapbox/mapbox-gl-native/blob/12cb77e5f22a05d66ad7fe30fd0bf62d50948214/platform/macos/src/MGLMapViewDelegate.h#L293

Create a new heading for “Managing the Style” or move this method up to the “Loading the Map” section.